README.md

```diff
@@ -1 +1,21 @@
-# css
\ No newline at end of file
+# CSS
+
+This is the CSS package for WARP.
+
+The package includes only resets for now.
+
+## Usage
+
+The CSS should be used directly from Eik. This is because we can alias releases by major versions.
+
+We publish 2 versions of presets. The minified that can be used in production and non-minified that can be used in development.
```

Comment:
resets? 😄

Reply:
ah damn it! What a hawk eye!! Thank you

Reply:
[Fixed](https://github.com/warp-ds/css/pull/2/commits/9b750a1229c035c576b928203b260b4444d4b86b)